content/en/continuous_integration/pipelines/awscodepipeline.md

```diff
@@ -30,9 +30,11 @@ Set up tracing on AWS CodePipeline to collect data about pipeline executions, an
 | Pipeline Visibility | Platform | Definition |
 |---|---|---|
 | [Partial retries][14] | Partial pipelines | View partially retried pipeline executions. |
-| [Running pipelines][15] | Running pipelines | View pipeline executions that are running. |
+| *[Running pipelines][15] | Running pipelines | View pipeline executions that are running. |
+| **Logs correlation | Logs correlation	| Correlate pipeline and job spans to logs and enable [job log correlation][16]. |
```

Comment:
```suggestion
| **Logs correlation | Logs correlation	| [Enable job log correlation](#enable-log-correlation) to correlate pipeline and job spans to logs. |
```

Reply:
We use the `Correlate pipeline and job spans to logs and enable job log correlation` in all the docs for other providers - imho we should be consistent.